network/src/peer_manager/conn_notifs_channel.rs

```diff
@@ -28,84 +28,54 @@ mod test {
     use libra_config::network_id::NetworkContext;
     use libra_network_address::NetworkAddress;
 
+    fn send_new_peer(sender: &mut Sender, peer_id: PeerId) {
+        sender
+            .push(
+                peer_id,
+                ConnectionNotification::NewPeer(
+                    peer_id,
+                    NetworkAddress::mock(),
+                    NetworkContext::mock(),
+                ),
```

Comment:
if you were to move this to another line `let cn = ...` followed by the sender.push, I bet this would be a lot more compact, same in a few other places.

Reply:
I'll see if I can do that!

Basically this stuff was copied so many places this should help for later